content/blog/benefits-of-the-platform-level-data-model-for-firmware-update-standard.md

```diff
@@ -0,0 +1,101 @@
+---
+title: Benefits of the Platform Level Data Model for Firmware Update Standard
+date: 2022-05-24T14:35:26.378Z
+author: François Donzé
+authorimage: /img/fdz-photoprofile.png
+tags:
+  - ilo-resful-api
+  - Redfish
+---
+## Introduction
+
+In 2016, the [Distributed Management Task Force](https://dmtf.org) published the first version of the Platform Level Data Model for Firmware Update Specification ([PLDM for FWUPD](https://www.dmtf.org/dsp/DSP0267)). This DMTF standard started to be implemented in the Redfish® service of HPE iLO 5 version 2.30.
+
+The best definition of PLDM for FWUPD is provided in the introduction paragraph of the [DSP0267](https://www.dmtf.org/sites/default/files/standards/documents/DSP0267_1.1.0.pdf) specification document:
+
+"*The Platform Level Data Model (PLDM) Firmware Update Specification defines messages and data structures for updating firmware or other code objects maintained within the firmware devices of a platform management subsystem. Additional functions related to the sequence of identifying and transferring the firmware, are also defined.*"
```

Comment:
I just noticed that you used quotation marks here and I don't believe you did in the previous blog post. It's a good idea to do so, though sometimes quotes are indicated simply through italitization. Can you take input from me here to put it into your other blog post?

Reply:
Done.